responses/05_configure-azure.md

```diff
@@ -74,7 +87,7 @@ jobs:
       - name: Configure webapp to use GitHub Packages
         if: success()
         run: |
-          {% raw %}az webapp config container set --docker-custom-image-name nginx --docker-registry-server-password ${{secrets.GITHUB_TOKEN}} --docker-registry-server-url https://docker.pkg.github.com --docker-registry-server-user ${{github.actor}} --name ${{ env.AZURE_WEBAPP_NAME }} --resource-group ${{ env.AZURE_RESOURCE_GROUP }} --subscription ${{secrets.AZURE_SUBSCRIPTION_ID}}{% endraw %}
+          {% raw %}az webapp config container set --docker-custom-image-name nginx --docker-registry-server-password ${{secrets.PACKAGE_PAT}} --docker-registry-server-url https://docker.pkg.github.com --docker-registry-server-user ${{github.actor}} --name ${{ env.AZURE_WEBAPP_NAME }} --resource-group ${{ env.AZURE_RESOURCE_GROUP }} --subscription ${{secrets.AZURE_SUBSCRIPTION_ID}}{% endraw %}
```

Comment:
@hectorsector 
Why not to use GitHub Action [Azure/appservice-settings@v1](https://github.com/Azure/webapps-deploy) to [Configure web app private registry credentials](https://github.com/Azure/webapps-deploy#configure-web-app-private-registry-credentials) instead? 🤔
This is working fine and I think it's better solution, because this lab is all about GitHub Actions 🙂
```yaml
      - name: Configure webapp to use GitHub Packages
        uses: Azure/appservice-settings@v1
        with:
         app-name: ${{env.AZURE_WEBAPP_NAME}}
         app-settings-json: |
           [
               {
                   "name": "DOCKER_REGISTRY_SERVER_PASSWORD",
                   "value": "${{ secrets.PACKAGE_PAT }}",
                   "slotSetting": false
               },
               {
                   "name": "DOCKER_REGISTRY_SERVER_URL",
                   "value": "${{env.IMAGE_REGISTRY_URL}}",
                   "slotSetting": false
               },
               {
                   "name": "DOCKER_REGISTRY_SERVER_USERNAME",
                   "value": "${{ github.actor }}",
                   "slotSetting": false
               }
           ]
```

I think it's even better to only replace `azure/docker-login@v1` actions in [03_workflow-steps.md](https://github.com/githubtraining/continuous-delivery-azure/blob/master/responses/03_workflow-steps.md) with above.